xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/config/vue.config.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
   chainWebpack: config => {
     // Provided dependencies (that shouldn't be bundled).
     config.externals({
+      "vue": "vue",
```

Comment:
It is: https://cli.vuejs.org/guide/build-targets.html#library
And I had the occasion to test it during debugging :p

Normally in that case you don't have to specify vue as an external dependency, but doing so allowed me to gain more control about when the vue module is imported, so that it is loaded before the `XWikiLivedata.vue` file is imported by `Logic.js`, so I can use `Vue.use()` to initialize vue plugins.